src/main/java/io/deephaven/csv/reading/ReaderUtil.java

```diff
@@ -0,0 +1,93 @@
+package io.deephaven.csv.reading;
+
+import io.deephaven.csv.containers.ByteSlice;
+import io.deephaven.csv.tokenization.RangeTests;
+import io.deephaven.csv.util.MutableInt;
+
+public class ReaderUtil {
+    public static String[] makeSyntheticHeaders(int numHeaders) {
+        final String[] result = new String[numHeaders];
+        for (int ii = 0; ii < result.length; ++ii) {
+            result[ii] = "Column" + (ii + 1);
+        }
+        return result;
+    }
+
+    /**
+     * Trim whitespace from the front and back of the slice.
+     *
+     * @param cs The slice, modified in-place to have whitespace (if any) removed.
+     */
+    public static void trimWhitespace(final ByteSlice cs) {
```

Comment:
This becoming newly public, I'm going to complain about the name "whitespace", as that has a more strict definition, see `java.lang.Character#isWhitespace(char)`.